catalyst/contrib/data/nifti/reader.py

```diff
@@ -0,0 +1,39 @@
+from typing import Optional
+
+import nibabel as nib
```

Comment:
```suggestion
from catalyst.settings import SETTINGS
if SETTINGS.nifti_required:
    import nibabel as nib
```

byt this way, we could build the documentation even without the `nibabel` package :)